libs/vre/resource-editor/representations/src/lib/archive/archive.component.ts

```diff
@@ -58,8 +58,8 @@ export class ArchiveComponent implements OnChanges {
     }
   }
 
-  download(url: string) {
-    this._rs.downloadFile(url);
+  download() {
+    this._rs.downloadProjectFile(this.src.fileValue, this.attachedProject?.shortcode);
```

Comment:
for this and all other instances, why can't we add inside: _rs.downloadProjectFile a viariable:
const attachedProject = this._store ...
rather than passing it from the resource-representation.component, and then to every child component until we reach that service ?

Reply:
I thought resource were not available everywhere. Looks like it might be taken from the store.